src/test/three-components/TextureUtils-spec.ts

```diff
@@ -29,23 +28,36 @@ const EQUI_URL = assetPath('spruit_sunrise_2k.jpg');
 const HDR_EQUI_URL = assetPath('spruit_sunrise_2k.hdr');
 
 suite('TextureUtils', () => {
+  let renderer: WebGLRenderer;
+
+  suiteSetup(() => {
+    // The renderer can retain state, so these tests have the possibility of
+    // getting different results in different orders. However, our use of the
+    // renderer *should* always return its state to what it was before to avoid
+    // this kind of problem (and many other headaches).
+    renderer = new WebGLRenderer({canvas});
+    renderer.debug.checkShaderErrors = true;
+  });
+
+  suiteTeardown(() => {
+    // Ensure we free up memory from loading large environment maps:
+    Cache.clear();
+    renderer.dispose();
```

Comment:
What does disposing of the renderer do that Cache.clear() doesn't do already? Every time we recreate a renderer we get a lot of console spew that makes reading the test output harder.

Reply:
See my comment above, I linked to the relevant lines. It's a hatchet, not a scalpel, for sure, but profiles bare out that there is a fair amount of memory (megabytes worth) being held by references in the renderer's internal cache.